contentcuration/contentcuration/frontend/shared/mixins.js

```diff
@@ -53,9 +53,7 @@ const statusStrings = createTranslator('StatusStrings', {
 });
 
 export const validationStrings = createTranslator('ValidationStrings', {
-  message: 'Missing required information',
-  context:
-    'An error message displayed when some information required before publishing a channel is missing, for example when a resource has no license set.',
+  message: 'Required information is missing',
 });
```

Comment:
This whole validator is actually just wrong! It should have been:
```js
export const validationStrings = createTranslator('ValidationStrings', {
  msgId: {
    message: 'Missing required information',
    context:
    'An error message displayed when some information required before publishing a channel is missing, for example when a resource has no license set.',
  }
});
```

but it isn't even used anywhere, so I'm deleting it!